File: Agencia de Viajes/API Agencia/agencia-viajes/internal/models/Pais.go
// # Package models
//
// Define las estructuras de dominio utilizadas en toda la aplicacion
// Movent para representar las entidades de la base de datos.
package models

// Pais
//
// Representa un pais registrado en el sistema.
// Se utiliza como referencia geografica para ciudades,
// usuarios y destinos de viaje.
type Pais struct {
	ID     int    `json:"id"`
	Nombre string `json:"nombre"`
}

File: Aero Linea/Aerolinea.API/Aerolinea.API/Models/Pais.cs
namespace Aerolinea.API.Models
{
    /// <summary>
    /// Entidad que representa un pais del catalogo geografico del sistema.
    /// Sirve como referencia para ciudades, aeropuertos y datos de usuario.
    /// </summary>
    public class Pais
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
    }
}
